### ISSUE TITLE
Error for systems without elements

### ISSUE THREAD

**Issue by JFRudzinski** [nomad-coe/nomad-schema-plugin-run#3]

**Comment by JFRudzinski** [comment]
```
"normalizer":string"SoapNormalizer"
"section":string"system"
"g_index":string"200"
"key_error":string"'A'"
"event":string"could not read a system property"
"proc":string"Entry"
"process":string"process_entry"
"process_worker_id":string"2J4SnRKeRqqoedIhzX9BSg"
"parser":string"parsers/gromacs"
"step":string"SoapNormalizer"
"logger":string"nomad.processing"
"exception":string"Traceback (most recent call last): File "/home/jfrudzinski/work/soft/nomad/nomad/normalizing/normalizer.py", line 159, in __normalize_system return self._normalize_system(system, representative) File "/home/jfrudzinski/work/soft/nomad/nomad/normalizing/normalizer.py", line 84, in _normalize_system return self.normalize_system(system, is_representative) File "/home/jfrudzinski/work/soft/nomad/dependencies/normalizers/simulation/soap/soapnormalizer/normalizer.py", line 57, in normalize_system atoms = system.atoms.to_ase(raise_exp=True) File "/home/jfrudzinski/work/soft/nomad/dependencies/schema/simulation/run/runschema/system.py", line 307, in to_ase raise e File "/home/jfrudzinski/work/soft/nomad/dependencies/schema/simulation/run/runschema/system.py", line 295, in to_ase return Atoms( File "/home/jfrudzinski/miniconda3/envs/nomad_env_2023_39_new/lib/python3.9/site-packages/ase/atoms.py", line 208, in __init__ self.new_array('numbers', symbols2numbers(symbols), int) File "/home/jfrudzinski/miniconda3/envs/nomad_env_2023_39_new/lib/python3.9/site-packages/ase/symbols.py", line 20, in symbols2numbers numbers.append(atomic_numbers[s]) KeyError: 'A'"
"timestamp":string"2024-02-08 10:07.36"
"level":string"ERROR"
```

**Comment by JFRudzinski** [comment]
fixed by [atomistic #93](https://github.com/nomad-coe/atomistic-parsers/pull/93)
